src/util/malloc/library.rs

```diff
@@ -36,8 +36,8 @@ mod jemalloc {
 
 #[cfg(feature = "malloc_mimalloc")]
 mod mimalloc {
-    // MiMalloc 64K Page
-    pub const LOG_BYTES_IN_MALLOC_PAGE: u8 = 16;
+    // Normal 4K page accounting
+    pub const LOG_BYTES_IN_MALLOC_PAGE: u8 = 12;
```

Comment:
I think it would be better to match what other malloc libraries do:
```suggestion
    pub const LOG_BYTES_IN_MALLOC_PAGE: u8 = crate::util::constants::LOG_BYTES_IN_PAGE;
```

Reply:
Yes, that makes sense. Thanks!